app/js/components/manageApps/ManageApps.jsx

```diff
@@ -245,19 +242,28 @@ export default class ManageApps extends React.Component {
   }
 
   displayManageOwaButtons() {
-    const addonFile = document.getElementById('fileInput').files[0];
-    if (typeof(addonFile) !== 'undefined') {
-      this.setState((prevState, props) => {
-        return {
-          displayManageOwaButtons: true
-        };
-      });
-    } else {
-      this.setState((prevState, props) => {
-        return {
-          displayManageOwaButtons: false
-        };
-      });
+    if (this.state.files !== null) {
+      return (
+        <div className="manage-owa-btns">
+          <div className="btn-toolbar">
+            <button
+              id="upload-btn"
+              className="btn"
+              onClick={this.handleUpload}>
+              <span
+                className="glyphicon glyphicon-upload"
+              /> Upload
+            </button>
+            <button
+              id="clear-btn"
+              className="btn"
+              onClick={this.handleClear}
+            >
```

Comment:
Looks good though I've noticed some of the HTML element closing angle brackets like on line 261 are pushed to a new line. Is this intentional?

Reply:
I think this is good. I do that always as well 😉

Reply:
@cnagadya  It was intentional. Helps me better visualize the code 😄 @andela-dbamidele , is it best practice to have it on the same line?